docs/api/standard_library_types.md

```diff
@@ -168,10 +168,7 @@ Pydantic supports the following numeric types from the Python standard library:
 ### `enum.IntEnum`
 
 * Validation: Pydantic checks that the value is a valid `IntEnum` instance.
-
-#### subclass of `enum.IntEnum`
-
-* Validation: checks that the value is a valid member of the integer enum;
+* Validation for subclass of `enum.IntEnum`: checks that the value is a valid member of the integer enum;
```

Comment:
```suggestion
* Validation for subclass of `enum.IntEnum`: checks that the value is a valid member of the integer enum.
```

Reply:
I think the `;` is correct, given there's the "see..." after.

Reply:
Ok, I've deleted my comment